ddl/job_table.go

```diff
@@ -454,15 +458,19 @@ func (d *ddl) MoveJobFromQueue2Table(force bool) error {
 			if err != nil {
 				return errors.Trace(err)
 			}
-			err = insertDDLJobs2Table(se, jobs, false)
-			if err != nil {
-				return errors.Trace(err)
-			}
-			if tp == generalWorker {
-				// general job do not have reorg info.
-				continue
-			}
 			for _, job := range jobs {
+				if inBootstrap && job.SchemaID == systemDBID {
+					// In bootstrap, we can ignore the internal DDL.
+					continue
+				}
+				err = insertDDLJobs2Table(se, false, job)
```

Comment:
We can filter the jobs in advance, so that the number of SQLs could be reduced to 1.

Reply:
does not matter, it should have no user ddl during the upgrade.

Reply:
I think it is ok, the ddl should not be a huge number, just more insert SQL is fine and the code is easy to understand.
BTW, code here is rare to use.